_install-and-configure/install-dashboards/debian.md

```diff
@@ -0,0 +1,129 @@
+{: .important}
+
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. Some example commands reference the `vi` text editor, but you may use any text editor available.
+{:.note}
+
+## Install OpenSearch Dashboards from a package
+
+1. Download the Debian package for the desired version directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}. The Debian package can be downloaded for both **x64** and **arm64** architectures.
+1. From the CLI, install using `dpkg`.
+   ```bash
+   # x64
+   sudo dpkg -i opensearch-dashboards-{{site.opensearch_version}}-linux-x64.deb
+   # arm64
+   sudo dpkg -i opensearch-dashboards-{{site.opensearch_version}}-linux-arm64.deb
+   ```
+1. After the installation completes, reload the systemd manager configuration.
+    ```bash
+    sudo systemctl daemon-reload
+    ```
+1. Enable OpenSearch as a service.
+    ```bash
+    sudo systemctl enable opensearch-dashboards
+    ```
+1. Start the OpenSearch service.
+    ```bash
+    sudo systemctl start opensearch-dashboards
+    ```
+1. Verify that OpenSearch launched correctly.
+    ```bash
+    sudo systemctl status opensearch-dashboards
+    ```
+
+### Fingerprint verification
+
+The Debian package is not signed. If you would like to verify the fingerprint, the OpenSearch Project provides a `.sig` file as well as the `.deb` package for use with GNU Privacy Guard (GPG).
+
+1. Download the desired Debian package.
+   ```bash
+   curl -SLO https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}-linux-x64.deb
+   ```
+1. Download the corresponding signature file.
+   ```bash
+   curl -SLO https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}-linux-x64.deb.sig
+   ```
+1. Download and import the GPG key.
+   ```bash
+   curl -o- https://artifacts.opensearch.org/publickeys/opensearch.pgp | gpg --import -
+   ```
+1. Verify the signature.
+   ```bash
+   gpg --verify opensearch-dashboards-{{site.opensearch_version}}-linux-x64.deb.sig opensearch-dashboards-{{site.opensearch_version}}-linux-x64.deb
+   ```
+
+## Install OpenSearch Dashboards from an APT repository
+
+APT, the primary package management tool for Debian–based operating systems, allows you to download and install the Debian package from the APT repository. 
+
+1. Import the public GPG key. This key is used to verify that the APT repository is signed.
+    ```bash
+    curl -o- https://artifacts.opensearch.org/publickeys/opensearch.pgp | sudo apt-key add -
+    ```
+1. Create an APT repository for OpenSearch:
```

Comment:
Global: Let's ensure that we consistently use a colon to introduce the code, as we've done here.